test/jdk/java/io/Serializable/class/NonSerializableTest.java

```diff
@@ -96,15 +99,17 @@ public Object[][] provider() {
         };
     }
 
-    @Test(dataProvider="provider")
-    public void test(String[] args) throws Exception {
+    @ParameterizedTest
+    @MethodSource("provider")
+    public void test(String[][] argArray) throws Exception {
```

Comment:
I recommend changing `provider` to return `List<String[]>` or `Object[]`, instead of hacking the test.